Azure/cloud-init.txt

```diff
@@ -0,0 +1,167 @@
+
+# Install tests dependencies
+apt-get install -y \
+	bc \
+	smem \
+	apache2-utils \
+	iperf
+
+# Install thin tools for devicemapper congifuration
+apt-get install -y thin-provisioning-tools
+
+# Configre the clear containers runtime using devicemapper as
+# storage driver
+mkdir -p /etc/systemd/system/docker.service.d/
+cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cc-runtime=/usr/bin/cc-runtime --default-runtime=cc-runtime
+EOF
+
+# Add devicemapper configuration
+mkdir -p /etc/docker
+
+mkdir -p /etc/lvm/profile
+cat << EOF | sudo tee /etc/docker/daemon.json
+{
+"storage-driver": "devicemapper",
+"storage-opts": [
+        "dm.directlvm_device=/dev/sdc",
+        "dm.thinp_percent=95",
+        "dm.thinp_metapercent=1",
+        "dm.thinp_autoextend_threshold=80",
+        "dm.thinp_autoextend_percent=20",
+        "dm.directlvm_device_force=true"
+]
+}
+EOF
+
+# Restart docker with devicemapper changes
+systemctl daemon-reload
+#systemctl restart docker
+
+# Clone tests/metrics repository
+git clone https://github.com/clearcontainers/tests.git
+
+# Update kernel
+echo deb http://archive.ubuntu.com/ubuntu/ xenial-proposed restricted main multiverse universe >> /etc/apt/sources.list
+apt-get update
+apt-get install -y linux-azure
+
+# Install Kubernetes and kubeadm
+apt-get install -y ebtables ethtool
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+cat << EOF > /etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+apt-get update
+apt-get install -y kubelet kubeadm kubectl
+
+# Create kubelet systemd service
+cat << EOF | sudo tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+[Service]
+Environment="KUBELET_KUBECONFIG_ARGS=--kubeconfig=/etc/kubernetes/kubelet.conf"
+Environment="KUBELET_SYSTEM_PODS_ARGS=--pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true --max-pods=500"
+Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
+Environment="KUBELET_DNS_ARGS=--cluster-dns=10.96.0.10 --cluster-domain=cluster.local"
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --container-runtime-endpoint=/var/run/crio.sock --runtime-request-timeout=30m --fail-swap-on=false"
+ExecStart=
+ExecStart=/usr/bin/kubelet \$KUBELET_KUBECONFIG_ARGS \$KUBELET_SYSTEM_PODS_ARGS \$KUBELET_NETWORK_ARGS \$KUBELET_DNS_ARGS \$KUBELET_EXTRA_ARGS
+EOF
+
+# Clone, build and install CRI-O
+add-apt-repository -y ppa:projectatomic/ppa
+add-apt-repository -y ppa:alexlarsson/flatpak
+add-apt-repository -y ppa:longsleep/golang-backports
+apt-get update
+apt-get install -y btrfs-tools git golang-go libassuan-dev libdevmapper-dev libglib2.0-dev libc6-dev libgpgme11-dev libgpg-error-dev libseccomp-dev libselinux1-dev pkg-config cri-o-runc runc skopeo libostree-dev skopeo-containers go-md2man
```

Comment:
@sameo I was missing make in my image. Let me verify if we need additional packages or it was a user error.